src/DataLayer/KeyMotorData/KeyMotorData.cpp

```diff
@@ -26,6 +26,7 @@ void KeyMotorData::setMotorZero(KeyMotor motorZero)
     emit motorActualSpeedReceived((motorZero_.vehicleVelocity() + motorOne_.vehicleVelocity()) / 2);
     emit motorBusVoltageReceived((motorZero_.busVoltage() + motorOne_.busVoltage()) / 2);
     emit motorBusCurrentReceived((motorZero_.busCurrent() + motorOne_.busCurrent()) / 2);
+    emit motorZeroAlive(motorZero.alive());
```

Comment:
Instead of emitting a separate `Alive` signal, the `motorZeroReceived` signal already contains the `alive` signal, so it makes more sense to use that in motorView.